usecases/backup/coordinator.go

```diff
@@ -0,0 +1,374 @@
+package backup
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/semi-technologies/weaviate/entities/backup"
+	"github.com/semi-technologies/weaviate/usecases/config"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/errgroup"
+)
+
+// Op is the kind of a backup operation
+type Op string
+
+const (
+	OpCreate  Op = "create"
+	OpRestore Op = "restore"
+)
+
+var (
+	errNoShardFound = errors.New("no shard found")
+	errCannotCommit = errors.New("cannot commit")
+)
+
+const (
+	_BookingPeriod      = 20
+	_TimeoutNodeDown    = 7 * time.Minute
+	_TimeoutQueryStatus = 5 * time.Second
+	_TimeoutCanCommit   = 8 * time.Second
+	_NextRoundPeriod    = 10 * time.Second
+	_MaxNumberConns     = 16
```

Comment:
Why are these prefixed with `_`?

Reply:
These are prefixed with _ to avoid exporting them outside the package